ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-routing.j2

```diff
@@ -1,24 +1,22 @@
 {# eos - IP Routing #}
-{% if ip_routing is defined and ip_routing is not none %}
-{%     if ip_routing == true %}
+{% if ip_routing is arista.avd.defined %}
+{%     if ip_routing is arista.avd.defined(true) %}
 !
 ip routing
 {%     else %}
 !
 no ip routing
 {%     endif %}
 {% endif %}
-{% if vrfs is defined and vrfs is not none %}
+{% if vrfs is arista.avd.defined %}
```

Comment:
As `arista.avd.defined` is included in `arista.avd.natural_sort`, if statement can be removed

Reply:
this check is redundant and should be removed